tensorrt_llm/_torch/pyexecutor/resource_manager.py

```
@@ -591,6 +595,21 @@ def append_to_kv_heads_per_layer(num_kv_heads_per_layer: List[int],
+                      self.get_cache_bytes_per_token())
+        layer_list = self.pp_layers
+        if layer_list:
+            layer_str = (f"{layer_list[0]}-{layer_list[-1]}"
+                         if len(layer_list) > 1 else str(layer_list[0]))
+        else:
+            layer_str = "(none)"
+        logger.info(
+            f"KV cache pool: dtype={self.dtype}, layers=[{layer_str}], "
+            f"num_layers={len(layer_list)}, "
+            f"pool_size={pool_bytes / (1 << 30):.2f} GiB "
+            f"({self.blocks_in_primary_pool} blocks × {self.tokens_per_block} tokens/block)")
```

Ruff (0.15.12)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)